tools/regs2groov.py

```diff
@@ -120,6 +127,17 @@ def parse_cmdline():
         help="How to represent names in the C++ code.",
     )
 
+    parser.add_argument(
+        "--bus",
+        type=str,
+        default="groov::mmio_bus<>",
+        help="Specify a custom bus interface for register groups to use.",
+    )
+
+    parser.add_argument(
+        "--includes", type=str, nargs='+', help="One or more include files to add."
```

Comment:
**issue**:  includes are optional, right? So use `nargs='*'` and `default=[]`?
See https://github.com/intel/compile-time-init-build/blob/main/tools/gen_str_catalog.py#L302